### PR TITLE
Ticket 83 release

### PR DESCRIPTION
**Pull request by sanjayguwaju**
sanjay digital clock added

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

File: index.html [hunk 1]
```diff
@@ -113,15 +113,15 @@ <h2>Contact Form Project</h2>
...
-                    <h2>ABC</h2>
-                    <p>SDFHdkfdh</p>
-                    <a href="./contact-form/contact-form.html">View project</a>
-                </div>
+        <!-- Start Digital Clock -->
+        <div class="card">
+            <img src="contact-form/asset/contact.jpg" alt="new_project screenshot">
+            <div class="card-content">
+                <h2>Digital Clock</h2>
+                <p>Shows you the recent time digitally.</p>
```
Comment (binaya-batas) [inline index.html:120]: Please remove the start digital clock comment.

**Review by binaya-batas — CHANGES_REQUESTED** [review]
Please make the changes.

File: index.html (hunk 1 above)
Comment (sanjayguwaju) [reply index.html:120]: changes made as per requested.

**Review by binaya-batas — APPROVED** [review]

**Review by ghost — APPROVED** [review]
ok

**Pull request merged**